features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -1992,6 +1993,21 @@ CREATE TRIGGER SCOPE_TRIG NO CASCADE
             END
 /
 
+CREATE TABLE APP_ROLE_ASSOCIATION (
+    ROLE_ID VARCHAR(255) NOT NULL,
+    APP_ID CHAR(36) NOT NULL,
+    CONSTRAINT APP_ROLE_UNIQUE UNIQUE (ROLE_ID, APP_ID),
```

Comment:
Shall we make this the primary key, instead of unique key
Not having a primary key for table could cause replication issues in mysql.

Reply:
fixed